README.md

```diff
@@ -231,18 +231,23 @@ An opinionated roadmap to become an SRE (Concepts > Tools)
 * TLS
 * PKI
 * Signature
-   
-## Analysis
 
-* Core analysis loop
-* Correlation vs. causation
-* First principle
-* Five whys technique
-* _Incident management_
-  * How to address an incident (assess, mitigate, resolve)
-  * Incident roles
-  * How to write a postmortem
-  * 3C principles (Coordinate, Communicate, maintain Control)
+## Incident management
```

Comment:
I tend to disagree (but this whole repo is opinionated 😉) about the main philosophy you want to address (analysis inside incident management). In my opinion, it's not necessary to face an incident to apply those analysis techniques. For example, a few days ago I started with an intuition (e.g., "I feel like this might be wrong/optimized/make simpler/whatever") and using the five why techniques, I was able to confirm my intuition. I didn't wait for an actual incident.

Reply:
yeah, i think a majority of my analysis has been done regarding incidents recently but i just came from a very buggy environment.

and the understanding running applications / ability to reproduce issues locally / advice on dev setup etc have been big parts of my SRE responsibilities too, outside of incidents.

on board w/ the idea that analysis and incident management are peers

im on board with trashing this PR, really love what you've written 😄